base/MQ/CMakeLists.txt

```diff
@@ -52,10 +52,6 @@ target_include_directories(${target} PUBLIC
   ${ROOT_INCLUDE_DIR} # TClonesArray, TMessage, etc
```

Comment:
Just a heads-up: The usage of `${ROOT_INCLUDE_DIR}` should also be removed, but this is only possible from ROOT 6.20 upwards.

Reply:
Let's move that into a new PR?